RELEASE.txt

```diff
@@ -204,7 +204,7 @@ If you want to take matters into your own hands for some reason, follow the
 instructions detailed below to cut a Debian release yourself.
 
 - Tag the release as per instructions above.
-- git checkout debian
+- git switch debian
```

Comment:
For now, leave this file alone; we will remove the Debian instructions soon enough, and the `debian` branch doesn't exist.